docs/source/quick_start.rst

```diff
@@ -6,170 +6,15 @@ the box there is support for pulling data from a multitude of
-   rpms_id: the field name of the RPMS ID.
-
-
-3. Encrypt the `lochness.json` by running::
-
-   bash 1_encrypt_command.sh
-
-This encryption step creates a copy of encrypted keyrings to
-`/data/lochness_root/.lochness.enc`. To protect the sensitive keyring
-information in json, remove the `lochness.json` after running the encryption.
-
-You can still extract keyring structure without sensitive information by running::
-
-   lochness_check_config.py -ke /data/lochness_root/.lochness.enc
-   
-
-4. Set up REDCap Data Entry Trigger if using REDCap. Please see below 
-   "REDCap Data Entry Trigger capture" section.
-
-
-5. Edit Personally identifiable information mapping table. Please seee below
-   "Personally identifiable information removal from REDCap and RPMS data"::
-
-    `/data/lochness_root/pii_convert.csv`
-
-
-5. Run the `sync.py` or use the example command in `2_synch_command.sh`::
-
-   bash 2_sync_command.sh
-
-
-REDCap Data Entry Trigger capture
----------------------------------
-If your sources include REDCap and you would like to configure lochness to 
-only pull new REDCap data, "Data Entry Trigger" needs to be set up in REDCap.
-
-In REDCap,
-- "Project Setup"
-  - "Enable optional modules and customizations"
-    - "Additional customizations"
-      - check "Data Entry Trigger" and give address of the server including
-        the port number. eg) http://pnl-t55-7.partners.org:9999
-
-
-In order to use this functionality, the server where lochness is installed
-should be able to receieve HTTP POST signal from REDCap server. Which means it
-has to be either
-
-- lochness server is inside the same firewall as REDCap server.
-    Or
-- lochness server has a open port that could listen to the REDCap POST signal.
-
-
-After setting the "Data Entry Trigger" on REDCap settings, run below to update
-the `/data/data_entry_trigger_db.csv` real-time::
-
-    # please specify the same port defined in the REDCap settings
-    listen_to_redcap.py --database_csv /data/data_entry_trigger_db.csv \
-                        --port 9999
-
-It would be useful to run `listen_to_redcap.py` in background, maybe inside a
-`gnu screen` so it runs continuously without interference.
-
-
-
-Personally identifiable information removal from REDCap and RPMS data
-----------------------------------------------------------------------
-A path of csv file can be provided, which has information about how to process
-each PII fields. 
-
-For example::
-
-    #/data/personally_identifiable_process_mappings.csv
-
-    pii_label_string | process
-    -----------------|---------------
-    address          | remove
-    date             | change_date
-    phone_number     | random_number
-    patient_name     | random_string
-    subject_name     | replace_with_subject_id
-
-Any value from the field, with names that match to `pii_label_string` rows,
-the labelled **PII processing method** will be used to process the raw values
-to remove or replace the PIIs.
```

Comment:
I understand that you have totally removed the above and decided not to hyperlink to the ones included in README.md.

Reply:
Link just added, Thanks